osbs/http.py

```diff
@@ -43,15 +43,19 @@ def __init__(self, verbose=False):
         self.verbose = verbose
 
     def get(self, url, **kwargs):
+        kwargs.setdefault('timeout', HTTP_REQUEST_TIMEOUT)
```

Comment:
All of those call `self.request`, it would be best to be set there

Reply:
yeah I've just realized it now :)